packages/vercel-edge/src/transports/types.ts

```diff
@@ -1,8 +0,0 @@
-import type { BaseTransportOptions } from '@sentry/core';
-
-export interface VercelEdgeTransportOptions extends BaseTransportOptions {
```

Comment:
This is defined and referenced from https://github.com/getsentry/sentry-javascript/blob/develop/packages/vercel-edge/src/transports/types.ts#L3 instead of this file. Probably was refactored at some point and missed during cleanup